frontend/src/components/user/UserTable.vue

```diff
@@ -103,8 +106,8 @@ const selection: Ref<UserAccessRequest | undefined> = ref(undefined);
         </Button>
       </template>
     </Column>
-    <Column
-      v-if="!revocation && authzStore.canNavigate(NavigationPermission.HOUSING_USER_MANAGEMENT_ADMIN)"
+    <!-- <Column
+      v-if="!requestTable && authzStore.canNavigate(NavigationPermission.HOUSING_USER_MANAGEMENT_ADMIN)"
       field="approve/deny"
```

Comment:
Is this commented out Column required as a comment? Or can it be removed?

Reply:
removed